src/timer.h

```diff
@@ -24,6 +24,9 @@ struct Timer {
     u8 resetsrc;
     u16 timer;
     u32 val;
+#if HAS_EXTENDED_AUDIO
+    u16 duration;
+#endif
     enum TimerType type;
     u8 padding_1[3];  //Needed to ensure structure is word aligned
```

Comment:
You may need to reduce padding_1 to 1 bytes as you are using additional 2 bytes. In other words, you don't need to add #if-def around this variable as anyway we are padding.

Reply:
thanks!